spec/features/users/merchant/merchant_update_spec.rb

```diff
@@ -11,15 +11,16 @@
       within "#item-#{item.id}" do
         click_link "edit"
       end
+
       expect(current_path).to eq(edit_item_path(item))
 
-      fill_in "thumbnail", with: "new_thumb"
+      fill_in "Price",	with: "11"      
```

Comment:
This is no longer doing what the test says by editing the image, its editing the price...

Reply:
If this is for #76 then it would be nice to know that. Its the only card that talks about editing an item.

Reply:
If assert the value of a new image you can just delete the image (user story says image does not have to be preset) or you can fill it in with `no_img.jpg` and expect for that.